pgcli/pgcompleter.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, smart_completion=True):
 
         self.special_commands = []
         self.databases = []
-        self.dbmetadata = {}
+        self.dbmetadata = {'tables': {}, 'functions': {}}
```

Comment:
I found the current nested approach useful in a couple spots:  like [here](https://github.com/amjith/pgcli/pull/155/files#diff-36fcb873925169e72e8c5ca4e8a8a581R91), where I loop over dbmetadata.keys(), and the `populate_schema_objects` method, which takes an argument like 'tables' or 'functions'. Neither of these are particularly important though, and could be pretty trivially replaced with explicit reference to separate dicts.

I think most of the `metadata = self.dbmetadata['tables']` stuff is just an unconcious/premature optimization on my part to pull dictionary lookups outside of loops. That pattern could stay or go independently of the dictionary storage.